astropy/modeling/bounding_box.py

```diff
@@ -20,7 +19,9 @@
 __all__ = ["ModelBoundingBox", "CompoundBoundingBox"]
 
 
-_BaseInterval = namedtuple("_BaseInterval", "lower upper")
+class _BaseInterval(NamedTuple):
```

Comment:
@WilliamJamieson, perhaps for a future PR, but I think that `_BaseInterval` can be merged with `_Interval`